CameraOverlay/tests/orchestrator_test.py

```diff
@@ -0,0 +1,38 @@
+"""Test Script For Orchestrator Class"""
```

Comment:
You might want to split this file into another PR

Reply:
We already have this in the master branch? I'm confused.

Reply:
@maanika  You can run `git merge master` to pull in new changes.